aspnetcore/signalr/introduction-signalr-core.md

```diff
@@ -0,0 +1,60 @@
+
+SignalR for ASP.NET Core:
+
+* Handles connection management automatically.
+* Enables broadcasting messages to all connected clients simultaneously. For example, a chat room.
+* Enables sending messages to specific clients or groups of clients.
+* Is open-sourced at [GitHub](https://github.com/aspnet/SignalR).
+* Scales nicely using [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/), SQL Server, or [Redis](http://redis.io).
+
+The connection between the client and server is persistent, unlike an HTTP connection. 
+
+## Transports and fallbacks
+
+ SignalR is an abstraction over some of the transports that are required to do real-time work between client and server. A SignalR connection starts as HTTP. SignalR uses the [WebSocket transport](https://tools.ietf.org/html/rfc7118) when available, and falls back to other transports when it's not available. WebSocket is the ideal transport for SignalR. This is because it makes the most efficient use of server memory, has the lowest latency, and has the most underlying features, such as full duplex communication between client and server. However, it also has the most stringent requirements: WebSocket requires the server to be using Windows Server 2012 or Windows 8, and .NET Framework 4.5, or later versions of these. If these requirements are not met, SignalR will attempt to use other transports to make its connections. The important thing to note is that SignalR automatically assigns a transport per connection for you automatically, though you can override this functionality.
+
+## Hubs and Connections
+
+The SignalR API contains two models for communicating between clients and servers: Hubs and Persistent Connections. The Hubs API covers the vast majority of scenarios the average ASP.NET developer needs.
+
+a layer that provides a raw socket-like API on top of Long Polling/Server-Sent Events/WebSockets, and as the layer on which SignalR is built
+
+A Connection represents a simple endpoint for sending single-recipient, grouped, or broadcast messages. The Persistent Connection API gives the developer direct access to the low-level communication protocol for SignalR. 
+
+A Hub is a high-level pipeline built upon the Connection API that allows your client and server to call methods on each other directly. SignalR handles the dispatching across machine boundaries as if by magic, allowing clients to call methods on the server as easily as local methods, and vice versa. Using a Hub allows you to pass strongly typed parameters to methods, enabling model binding. SignalR provides two built-in hub protocols: a text protocol based on JSON and a binary protocol based on MessagePack. Using  MessagePack generally creates smaller messages than when using JSON.
```

Comment:
~~Connection~~ Endpoint API. 

Also, might be good to link to http://msgpack.org/ when referencing MessagePack.

> Using  MessagePack generally creates smaller messages than when using JSON.

It's also worth calling out the trade-off: Older browsers (specifically those which do **not** support [XHR level 2](https://caniuse.com/#feat=xhr2)) cannot support the MessagePack protocol